packages/terra-worklist-data-grid/src/subcomponents/Row.jsx

```diff
@@ -98,7 +98,7 @@ function Row(props) {
   const columnIndexOffSet = hasRowSelection ? 1 : 0;
 
   const handleCellSelect = useCallback((selectionDetails) => {
-    if (hasRowSelection || selectionDetails.multiSelect) {
+    if (hasRowSelection || selectionDetails.isShiftKeyPressed) {
       if (onRowSelect) {
         onRowSelect(selectionDetails);
       }
```

Comment:
As we look to modularize the functionality for our upcoming enhancements, I am wondering if we should allow the WorklistDataGrid to handle this logic.  Moving the handler logic there removes the grid specific logic from the Row subcomponent.

Reply:
Point taken but this Jira did not change that logic. As a result, the suggestion you made should be handled in the Jira that creates the base data grid component.